game_eggs/steamcmd_servers/conan_exiles/egg-conan-exiles.json

```diff
@@ -35,18 +35,20 @@
             "description": "The ID corresponding to the game to download.",
             "env_variable": "SRCDS_APPID",
             "default_value": "443030",
-            "user_viewable": true,
+            "user_viewable": false,
             "user_editable": false,
-            "rules": "required|numeric|digits_between:1,7"
+            "rules": "required|numeric|digits_between:1,7",
+            "field_type": "text"
         },
         {
             "name": "XVFB Enable",
-            "description": "1 or 0",
+            "description": "MUST BE 1",
             "env_variable": "XVFB",
             "default_value": "1",
             "user_viewable": false,
             "user_editable": false,
-            "rules": "required|numeric|min:0|max:1"
+            "rules": "required|numeric|min:0|max:1",
```

Comment:
This can be: `required|numeric|in:1,0`

Reply:
this is from old egg. I dont change this. But i will change it now

Reply:
done